airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2698,6 +2753,49 @@ components:
           type: string
         memory_limit:
           type: string
+    # DB Migration
+    DbMigrationInfoItem:
+      type: object
+      required:
+        - migrationType
+        - migrationVersion
+        - migrationDescription
+      properties:
+        migrationType:
+          type: string
+        migrationVersion:
+          type: string
+        migrationDescription:
+          type: string
+        migrationState:
```

Comment:
> should this be an enum?

The backend passes in the display name of the state enum (e.g. the display name for `MISSING_FAILED` is `Failed (Missing)`). So a string seems move convenient.

> and should it be required?

No. I reuse the `DbMigrationRead` for both general migration and executed migration because they are very similar (in that the latter has a subset of fields of the former). The state does not exist for the executed migration.

Reply:
Actually I will change it to enum.